tests/handlers.rs

```diff
@@ -353,6 +353,68 @@ speculate! {
     }
 
     describe "Completion request" {
+        describe "when completing params" {
```

Comment:
could you add another test to verify object funcs?

Reply:
👍